ledger/sandbox-on-x/BUILD.bazel

```diff
@@ -576,6 +576,22 @@ server_conformance_test(
     ],
 )
 
+server_conformance_test(
```

Comment:
Can we merge this test with the one above under the `conformance-test-dev-features`? It takes a long time to set-up these tests

Reply:
👍 Good idea. These features are not mutually exclusive

Reply:
Ah actually no. One is running LF 1.15 and mine is 1.dev